appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">org.citra.citra.desktop</id>
+  <name>Citra</name>
+  <summary>Nintendo 3DS emulator</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <content_rating type="oars-1.0">
+    <content_attribute id="social-chat">moderate</content_attribute>
+  </content_rating>
+  <description>
+    <p>Citra is an experimental open-source Nintendo 3DS emulator/debugger written in C++. It is written with portability in mind, with builds actively maintained for Windows, Linux and macOS.</p>
+    <p>Citra emulates a subset of 3DS hardware and therefore is useful for running/debugging homebrew applications, and it is also able to run many commercial games! Some of these do not run at a playable state, but we are working every day to advance the project forward. (Playable here means compatibility of at least "Okay" on our game compatibility list.)</p>
+  </description>
+  <provides>
+    <binary>citra-qt</binary>
+    <id>org.citra.citra.desktop</id>
```

Comment:
You don't provide a `org.citra.citra.desktop` id, it's `org.citra.citra` which is already used by defining `id`.  Instead, you should use the old id (which is probably `citra.desktop`). This is mostly used by Software centers to merge two applications once the ID was updated to something else.